tests/e2e/.onBoardingPage.test.js

```diff
@@ -0,0 +1,48 @@
+async function validate_checkboxes(checkbox_selector, boxes_to_check) {
```

Comment:
Made this hidden since we cannot make sure this gets run first... It is a little sad haha